Review of EMT Standalone Node docs (#43)
open-edge-platform/edge-microvisor-toolkit-standalone-node: standalone-node/docs/user-guide/Get-Started-Guide.md

# Get Started

The Edge Microvisor Toolkit Standalone Node uses the standard immutable build. You can can build your own bootable USB from source code, or use the downloadable ISO
image that can be flashed to a USB device and installed on edge nodes. It
installs the microvisor and Kubernetes to the edge node with the essential
functionality to run a single node cluster. The edge node will serve as both the
control and worker node. Additional worker nodes can be added to the cluster
through Kubernetes.

Future releases will enable standalone edge nodes to join an existing Edge
Management Framework backend, deployed on-prem or in the cloud to support scale
out and management of larger infrastructures. The Standalone Edge Node enables
you to quickly get an edge node up and running without deploying backend

> services, ready to deploy Kubernetes applicationsthrough `kubectl`, `helm`, or
> Kubernetes web dashboard.
> 
> >**Note**: The standalone edge node does not currently support the real-time version.

Reviewer: ```suggestion
services, ready to deploy Kubernetes applications through `kubectl`, `helm`, or
Kubernetes web dashboard.

> **Note**: The standalone edge node does not currently support the real-time version.
```

     ```

     > **Note:**  Providing proxy settings is optional if the edge node does not require them to access internet services.
  

>   - Unplug the attached bootable USB from developer system before proceeding with deployment.

Reviewer: ```suggestion
  - Unplug the attached bootable USB drive from developer system before proceeding with deployment.
```

## Standalone Node Deployment

> - Plug the created bootable USB pen drive into the standalone node machine.

Reviewer: ```suggestion
- Plug the created bootable USB drive into the standalone node machine.
```

> - Set the BIOS boot manager to boot from the USB pen drive.

Reviewer: ```suggestion
- Set the BIOS boot manager to boot from the USB drive.
```
